min-requirements.txt

```diff
@@ -1,3 +1,4 @@
 sqlite3,https://sqlite.org/2017/sqlite-autoconf-3170000.tar.gz -H sha256:a4e485ad3a16e054765baf6371826b5000beed07e626510896069c0bf013874c -X autotools -DCMAKE_POSITION_INDEPENDENT_CODE=On
 boost@1.72 -DCMAKE_POSITION_INDEPENDENT_CODE=On --build
 half,https://github.com/pfultz2/half/archive/1.12.0.tar.gz -X header -H sha256:0a08660b68abb176ebc2a0cdf8de46e3182a7f46c66443bb80dbfaaec98cf969 --build
+ROCmSoftwarePlatform/llvm-project-mlir@a24618b5694a4e7e1e60cec8a41dffa3e9993ba7 -DBUILD_FAT_LIBMLIRMIOPEN=1
```

Comment:
So due to PR #1233 these dependencies have been moved to requirements.txt, so this will need to be moved there as well.

Reply:
Done, thanks for reminding.